src/node-entry.ts

```diff
@@ -1,8 +1,3 @@
-/// <reference path="../typings/package.json.d.ts" />
-
-export * from './rollup/index';
-export * from './watch/index';
-
```

Comment:
Interesting. We were doing wild things here. So as long as nobody imported the classes in `watch/index.js` I guess we should be fine here. Nice catch!